src/Resources/js/views/mealGuest.ts

```diff
@@ -0,0 +1,44 @@
+export default class MealGuestView {
+    constructor() {
+        this.updateSlots();
+        setInterval(this.updateSlots, 3000);
+    }
+
+    private updateSlots() {
+        let $slotSelector = $('#invitation_form_slot');
+        if ($slotSelector.length < 1) {
+            return;
+        }
+
+        const date = $slotSelector.closest('.meal-guest').data('date');
```

Comment:
I guess you can get $slotSelector and date before and set it as parameters to the updateSlots callback. not sure if it has any performance effect, but you're not searching for the same data after each 3000ms.

Reply:
Make sense. Fixed.